packages/@lwc/engine-server/src/__tests__/fixtures/attribute-component-global-html/expected.html

```diff
@@ -11,7 +11,7 @@
       spellcheck="true"
       tabindex="-1"
       class="foo bar"
-      style="color: red; background: blue;"
+      style="color: red; background: blue"
```

Comment:
This test compares formatted HTML to formatted HTML, which I assume is the reason for this difference. I didn't change the test; I just ran `find . -type f | grep __tests__ | grep expected | xargs rm -f` and re-ran `yarn test` to generate the files.